cookbooks/aws-parallelcluster-platform/resources/fabric_manager/partial/_fabric_manager_install_rhel.rb

```diff
@@ -14,9 +14,14 @@
 
 action :install_package do
   package 'yum-plugin-versionlock'
-
-  package fabric_manager_package do
-    version fabric_manager_version
-    action %i(install lock)
+  bash "Install #{fabric_manager_package}" do
```

Comment:
Did you evaluate if adding `flush_cache({ before: true })` was enough? we're already using it in other places.

If this is not enough, did you try with `yum_package` resource, rather than manually writing the command?

Reply:
Yes we tried both without success.